src/app/profile/my-spaces/my-spaces-item-actions/my-spaces-item-actions.component.spec.ts

```diff
@@ -1,7 +1,7 @@
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
Is this being used?

Reply:
I remember removing them but missed it somehow during rebase.